star-dash/star-dash/Rendering/PlayerView/PlayerView.swift

```diff
@@ -35,6 +35,7 @@ class PlayerView {
         superview.addSubview(self.minimapView)
 
         self.controlView = ControlView(frame: superview.bounds, rotatedBy: rotation)
+        controlView.rotatedBy = rotation
```

Comment:
This could be initialised in `ControlView` itself during its init

Reply:
how to do this? overriding UIView's init would cause a error where its not possible to override